source/clear-linux/guides/maintenance/repair-failed-install.rst

```diff
@@ -0,0 +1,71 @@
+.. _repair-failed-install:
+
+Repair a failed installation of |CL-ATTR|
+#########################################
+
+This guide explains how to repair a failed installation of |CL| using a live
+installer image on a USB. It's assumed you already installed |CL| on a target system, but your operating system does not boot or function properly.
+
+Prerequisites
+*************
+
+* :ref:`download-cl-installer-image`
+
+Boot installer image to repair target system
+********************************************
+
+#. Boot the |CL| installer image.
+
+   .. warning::
+
+      Only launch the installer to enable command-line access to the target system. Do not proceed with the installation.
+
+.. include:: ../bare-metal-install-beta/bare-metal-install.rst
+   :start-after: install-on-target-start:
+   :end-before: install-on-target-end:
+
+#. Press :kbd:`CTL+ALT+F2` to launch the console.
+
+#. Log in as `root` and set a password.
+
+#. Assure the system is connected to the network.
+
+#. Mount the system’s root partition.
+
+   #. To find the root partition, run:
+
+      .. code-block:: bash
+
+         lsblk
+
+      We find root is `/dev/sda3/`.
+
+   #. Next, mount root.
+
+      .. code-block:: bash
+
+         mount /dev/sda3 /mnt
+
```

Comment:
I suggest adding a 3rd verification step to have the reader check that what is mount in `/mnt` is actually their rootfs on the machine. This could be checking for a number of things that indicate the right partition is mounted . Maybe checking `/mnt/etc/machine-id` for example. 

If the wrong partition got mounted, the reader might have to wait an unnecessarily long time as swupd downloads a _whole_ installation in the wrong place. Or a worse scenario: the wrong partition containing another OS is accidentally overwritten.

Reply:
or you can inspect /mnt/usr/lib/os-release as well

Reply:
Per @puneetse's suggestion, let's add this line:

# Verify that you mounted the correct root partition by checking the list of bundles installed on it by looking in /usr/share/clear/bundles.  

$ sudo ls /usr/share/clear/bundles

Reply:
Couldn't you also just run this commands to verify?
`sudo blkid /dev/sda3` 
Then check the UUID. 

The lack of a default `/etc/fstab` may throw off some Linux users. 

@bktan8, I leave it up to you.

Reply:
> or you can inspect /mnt/usr/lib/os-release as well

good suggestion! I was thinking that if a repair is needed that `/usr` may be in an unknown state and that file could be missing which is why I said the file in `etc`. The document could offer multiple suggestion though.



> Per @puneetse's suggestion, let's add this line:
> 
> Verify that you mounted the correct root partition by checking the list of bundles installed in /usr/share/clear/bundles. swupd will verify and fix the system against this list of bundles.
> $ sudo ls /usr/share/clear/bundles

See my note above. Same thought. 

> Couldn't you also just run this command to verify?
> `sudo blkid /dev/sda3`
> Then check the UUID.

This wouldn't be unique enough. It can indicate it is *a Linux* partition but wouldn't be definitive enough to say that it is a Clear Linux root partition.